src/fetch_receipts.py

```diff
@@ -174,4 +204,5 @@ def run(target, limit=None):
     args = parser.parse_args()
 
     # run
-    run(args.target, args.limit)
+    result = run(args.target, args.limit)
+    print_report(result)
```

Comment:
What about `run()` calling `print_report()` at the end of its execution? Or is it an issue for multiprocessing (knowing when all the processes are done)?